SilasKenneth/fast-food-fast-ui
# [Finishes #160132538] Create a page to add new fast food items

#### What this pull request does
Creates the page on the admin dashboard where an admin can add a new item
##### Description of task to be completed
 Create a page where an admin user can add fast food items
    -  Add links items orders account and log out
##### Relevant pivotal tracker story
[#160132538](https://www.pivotaltracker.com/n/projects/2194200/stories/160132538)
##### How to test
  - Clone the repo ``git clone https://github.com/SilasKenneth/fast-food-fast-ui.git``
  -  Go to the fast-food-fast-ui directory and head to UI folder, ```git checkout the ft-admin-add-item-160132538```
  - Open the additem.html file in your browser and that should work

#### Screenshot
![Add item](https://user-images.githubusercontent.com/11555354/44911069-3428f380-ad2e-11e8-85c0-deb8dfa6f31a.png)
